docs/user/ppl/index.rst

```diff
@@ -48,111 +48,83 @@ The query start with search command and then flowing a set of command delimited
+  `ml command <cmd/ml.rst>`_:                                     2.5                 stable (since 2.5)        Apply machine learning algorithms to analyze data.
+  `kmeans command <cmd/kmeans.rst>`_                              1.3                 stable (since 1.3)        Apply the kmeans algorithm on the search result returned by a PPL command.
+  `ad command <cmd/ad.rst>`_                                      1.3                 deprecated (since 2.5)    Apply Random Cut Forest algorithm on the search result returned by a PPL command.
+  `describe command <cmd/describe.rst>`_                          2.1                 stable (since 2.1)        Query the metadata of an index.
+  `explain command <cmd/explain.rst>`_                            3.1                 stable (since 3.1)        Explain the plan of query.
+  `show datasources command <cmd/showdatasources.rst>`_           2.4                 stable (since 2.4)        Query datasources configured in the PPL engine.
+  ==============================================================  ==================  ========================  ==============================================================================================
+
+  - `Syntax <cmd/syntax.rst>`_ - PPL query structure and command syntax formatting
 
-  - `ad command <cmd/ad.rst>`_
-
-  - `append command <cmd/append.rst>`_
-
-  - `appendcol command <cmd/appendcol.rst>`_
-
-  - `bin command <cmd/bin.rst>`_
-
-  - `dedup command <cmd/dedup.rst>`_
-
-  - `describe command <cmd/describe.rst>`_
-
-  - `eval command <cmd/eval.rst>`_
-
-  - `eventstats command <cmd/eventstats.rst>`_
-
-  - `expand command <cmd/expand.rst>`_
-
-  - `explain command <cmd/explain.rst>`_
-
-  - `fields command <cmd/fields.rst>`_
-
-  - `fillnull command <cmd/fillnull.rst>`_
-
-  - `flatten command  <cmd/flatten.rst>`_
-
-  - `grok command <cmd/grok.rst>`_
-
-  - `head command <cmd/head.rst>`_
-  
-  - `join command  <cmd/join.rst>`_
-
-  - `kmeans command <cmd/kmeans.rst>`_
-
-  - `lookup command <cmd/lookup.rst>`_
-
-  - `ml command <cmd/ml.rst>`_
-
-  - `multisearch command <cmd/multisearch.rst>`_
-
-  - `parse command <cmd/parse.rst>`_
-
-  - `patterns command <cmd/patterns.rst>`_
-
-  - `rare command <cmd/rare.rst>`_
-
-  - `rename command <cmd/rename.rst>`_
-
-  - `regex command <cmd/regex.rst>`_
-
-  - `rex command <cmd/rex.rst>`_
-
-  - `search command <cmd/search.rst>`_
-
-  - `show datasources command <cmd/showdatasources.rst>`_
-
-  - `sort command <cmd/sort.rst>`_
-
-  - `spath command <cmd/spath.rst>`_
-
-  - `stats command <cmd/stats.rst>`_
-
-  - `subquery (aka subsearch) command <cmd/subquery.rst>`_
-
-  - `reverse command <cmd/reverse.rst>`_
-
-  - `table command <cmd/table.rst>`_
-  
-  - `timechart command <cmd/timechart.rst>`_
+* **Functions**
 
-  - `top command <cmd/top.rst>`_
+  - `Aggregation Functions <functions/aggregation.rst>`_
```

Comment:
question: Should we have descriptions or some further info for these too?

Not clear how to find e.g. the `SUM` function (is it an aggregation function or a math function?).